touchnet/bin/convert_dcp_to_hf.py

```
@@ -51,6 +67,19 @@ def save_pretrained(args: CkptConverterConfig):
         logger.info(f"Saving the model to {ckpt_hf_dir}")
         model.save_pretrained(ckpt_hf_dir)
 
+        try:
+            feature_extractor = AutoFeatureExtractor.from_pretrained(args.tokenizer_model)
+            processor_component_to_save = feature_extractor
+        except ValueError:
+            try:
+                processor = AutoProcessor.from_pretrained(args.tokenizer_model)
+                processor_component_to_save = processor
+            except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)